osu.Game/Screens/Menu/Disclaimer.cs

```diff
@@ -190,7 +190,7 @@ private string getRandomTip()
             {
                 "You can press Ctrl-T anywhere in the game to toggle the toolbar!",
                 "You can press Ctrl-O anywhere in the game to access options!",
-                "All settings are dynamic and take effect in real-time. Try changing the skin while playing!",
+                "All settings are dynamic and take effect in real-time. Try pausing and changing the skin while playing!",
```

Comment:
that's not really proper English

Reply:
of course, that's what the **maybe** is for, I'm pointing that there's kind of an oxymoron here that shouldn't really exist in such place. (or anywhere)

Reply:
I don't really think this needed to change.

Reply:
maybe not... I'm just super not looking forward to potentially having someone claim that the tip is wrong because "settings menu can't be opened during gameplay"